### PR TITLE
Inventory fixes

### PR DESCRIPTION
**Pull request by boy0001**
(mostly affected win 10)
Creative can execute any transaction
Creative can try to drop any item
Send current item on invalid transaction

### CODE REVIEW

**Comment by Creeperface01** [comment]
Awesome, idk are @Pub4Game @MagicDroidX alive? :D

File: src/main/java/cn/nukkit/Player.java [hunk 1]
```diff
@@ -2622,15 +2622,19 @@ public void handleDataPacket(DataPacket packet) {
                         break;
                     }
 
-                    item = this.inventory.contains(dropItem.item) ? dropItem.item : this.inventory.getItemInHand();
+                    boolean creative = this.isCreative();
+
+                    item = (creative || this.inventory.contains(dropItem.item)) ? dropItem.item : this.inventory.getItemInHand();
```
Comment (Pub4Game) [inline src/main/java/cn/nukkit/Player.java]: Just do `this.isCreative()`

File: src/main/java/cn/nukkit/Player.java [hunk 2]
```diff
@@ -2622,15 +2622,19 @@ public void handleDataPacket(DataPacket packet) {
...
                     PlayerDropItemEvent dropItemEvent = new PlayerDropItemEvent(this, item);
                     this.server.getPluginManager().callEvent(dropItemEvent);
                     if (dropItemEvent.isCancelled()) {
                         this.inventory.sendContents(this);
                         break;
                     }
 
-                    this.inventory.removeItem(item);
```
Comment (Pub4Game) [inline src/main/java/cn/nukkit/Player.java]: same thing

**Comment by boy0001** [comment]
@Pub4Game 
Okay, changed it.

**Comment by File14** [comment]
Merge? I really want these fixes.

**Pull request merged**